Library/Formula/dlite.rb

```diff
@@ -0,0 +1,39 @@
+class Dlite < Formula
+  desc "The simplest way to use Docker on OSX"
```

Comment:
This doesn't really tell you what it does, though? Could you elaborate a bit there.